db/GCS.go

```diff
@@ -0,0 +1,97 @@
+package db
+
+import (
+	"context"
+	"io/ioutil"
```

Comment:
"io/ioutil" is deprecated: As of Go 1.16, the same functionality is now provided by package io

Reply:
Swapped according to this answer:
```
ioutil.ReadAll -> io.ReadAll
ioutil.ReadFile -> os.ReadFile
ioutil.ReadDir -> os.ReadDir
// others
ioutil.NopCloser -> io.NopCloser
ioutil.TempDir -> os.MkdirTemp
ioutil.TempFile -> os.CreateTemp
ioutil.WriteFile -> os.WriteFile
```

https://stackoverflow.com/questions/75206234/for-go-ioutil-readall-ioutil-readfile-ioutil-readdir-deprecated